app/channels/conversation_channel.rb

```diff
@@ -0,0 +1,52 @@
+class ConversationChannel < ApplicationCable::Channel
+  def subscribed
+    conversation_id = params[:conversation_id]
+    reject unless conversation_id.present?
+    stream_from("Conversation-#{conversation_id}")
+  end
+
+  def speak(data)
+    @data = data
+    validate_data
+    save_message
+    broadcast_message
+  end
+
+  private
+
+  def validate_data
+    validate_user
+    validate_conversation
+    validate_message
+  end
+
+  def validate_user
+    @sender = User.find_by(id: @data['sender_id'])
+    connection.transmit identifier: params, error: 'The user was not found.' unless
+      @sender
+  end
+
+  def validate_conversation
+    @conversation = Conversation.find_by(id: @data['conversation_id'])
+    connection.transmit identifier: params, error: 'The conversation was not found.' unless
+      @conversation
+  end
+
+  def validate_message
+    @message = @data['message']
+    connection.transmit identifier: params, error: 'The message must not be blank.' unless
+      @message
+  end
+
+  def save_message
+    Message.create(conversation: @conversation, sender: @sender, content: @message)
+  end
+
+  def broadcast_message
```

Comment:
I would do this as a callback in the message creation and using the specific channel calling the method broadcast_to 

https://stackoverflow.com/questions/41806862/difference-between-broadcast-broadcast-to-and-broadcast-for-in-rails-5